docs/introduction/quick-start.md

```diff
@@ -43,9 +43,12 @@ Confirm installation:
 
 ```
 $ tendermint version
-0.23.0-dev
+0.23.0
```

Comment:
we should probably remove this altogether so we dont have to update with each release and instead below say "the version should match that of the latest release"

Reply:
this was done somewhere. `-dev` removed and "NOTE" added right below, with link to releases page